model/task/task.go

```diff
@@ -1745,6 +1740,108 @@ func (t *Task) Archive() error {
 	return nil
 }
 
+func ArchiveMany(tasks []Task) error {
+	if len(tasks) == 0 {
+		return nil
+	}
+	// add execution tasks of display tasks passed in, if they are not there
+	execTaskMap := map[string]*Task{}
+	for _, t := range tasks {
+		execTaskMap[t.Id] = &t
+	}
+	additionalTasks := []string{}
+	for _, t := range tasks {
+		for _, et := range t.ExecutionTasks {
+			if execTaskMap[et] == nil {
+				additionalTasks = append(additionalTasks, t.ExecutionTasks...)
```

Comment:
if one execution task doesn't exist you add all execution tasks--is it ever possible for some to be in here but not others? can this have a comment?